packages/form-builder/src/App.scss

```diff
@@ -97,15 +97,16 @@
     //display: none; /* CSS Hack to hide root level inserter @todo Move to a JS or config solution. */
 }
 
-[data-type="custom-block-editor/section"] > div > .block-editor-inner-blocks > .block-editor-block-list__layout,
-[data-type="custom-block-editor/donor-info"] > div > .block-editor-inner-blocks > .block-editor-block-list__layout,
-[data-type="custom-block-editor/donation-amount"] > div > .block-editor-inner-blocks > .block-editor-block-list__layout,
-[data-type="custom-block-editor/payment-details"] > .block-editor-inner-blocks > .block-editor-block-list__layout,
-[data-type="custom-block-editor/custom-fields"] > .block-editor-inner-blocks > .block-editor-block-list__layout {
-    display: flex;
-    gap: 20px;
-    flex-direction: column;
-    margin-bottom: 40px;
+[data-type="custom-block-editor/section"],
+[data-type="custom-block-editor/donor-info"],
+[data-type="custom-block-editor/donation-amount"],
+[data-type="custom-block-editor/payment-details"] {
+    & > div > .block-editor-inner-blocks > .block-editor-block-list__layout {
+        display: flex;
+        gap: 24px;
+        flex-direction: column;
+        margin-bottom: 40px;
+    }
 }
```

Comment:
This can be simplified to generate less CSS:
```css
:is([data-type="custom-block-editor/section"], [data-type="custom-block-editor/donor-info"], [data-type="custom-block-editor/donation-amount"], [data-type="custom-block-editor/payment-details"]) > div > .block-editor-inner-blocks > .block-editor-block-list__layout {
        display: flex;
        gap: 24px;
        flex-direction: column;
        margin-bottom: 40px;
}
```

Double-check me, but I believe that should work. The difference it that doing it the way you have it, the SCSS will  compile to:
```css
[data-type="custom-block-editor/section"] > div > .block-editor-inner-blocks > .block-editor-block-list__layout, [data-type="custom-block-editor/donor-info"] > div > .block-editor-inner-blocks > .block-editor-block-list__layout, [data-type="custom-block-editor/donation-amount"] > div > .block-editor-inner-blocks > .block-editor-block-list__layout, [data-type="custom-block-editor/payment-details"] > div > .block-editor-inner-blocks > .block-editor-block-list__layout {
  display: flex;
  gap: 24px;
  flex-direction: column;
  margin-bottom: 40px;
}
```

Using `:is` will compile to:
```css
:is([data-type="custom-block-editor/section"], [data-type="custom-block-editor/donor-info"], [data-type="custom-block-editor/donation-amount"], [data-type="custom-block-editor/payment-details"]) > div > .block-editor-inner-blocks > .block-editor-block-list__layout {
  display: flex;
  gap: 24px;
  flex-direction: column;
  margin-bottom: 40px;
}
```